stream_alert_cli/terraform/monitoring.py

```diff
@@ -69,4 +86,13 @@ def generate_monitoring(cluster_name, cluster_dict, config):
             'kinesis_alarms_enabled': True
         })
 
+    # Add support for custom settings for tweaking alarm thresholds, eval periods, and periods
+    # Note: This does not strictly check for proper variable names, since there are so many.
+    #       Instead, Terraform will error out if an imporper name is used.
+    #       Also, every value in these settings should be a string, so cast for safety.
+    if monitoring_config.get('settings', {}):
+        for setting_name, setting_value in monitoring_config['settings'].iteritems():
```

Comment:
You don't need the top-level `if` statement: `.iteritems()` will not return anything for an empty dict.

```python
for setting_name, setting_value in monitoring_config.get('settings', {}).iteritems():
```